packages/f-checkout/src/constants.js

```diff
@@ -13,3 +13,14 @@ export const TENANT_MAP = {
     'it-IT': 'it',
     'nb-NO': 'no'
 };
+
+export const FULFILLMENT_TIMES = [
+    {
+        from: 'beginning',
```

Comment:
`from` and `to` normally will contain a datetime. I'd stick to that, to make the test look as real as possible (e.g. `"from": "2020-01-01T00:00+00:00"`)